app/controllers/miq_policy_controller.rb

```diff
@@ -358,7 +358,6 @@ def peca_get_all(what, get_view)
     @showlinks           = true
     @lastaction          = "#{what}_get_all"
     @force_no_grid_xml   = true
-    @gtl_type            = "list"
```

Comment:
application_controller `perpage_key` still assumes `@gtl_type` exists (but we can just update it to assume list)